sonar-vbnet-plugin/src/main/resources/documentation.md

```diff
@@ -16,15 +16,30 @@ Discover and update the VB.NET-specific [properties](/analysis/analysis-paramete
 
 To analyze tool-generated code (e.g. WCF code generated by `SvcUtil.exe`) for a specific VB.NET project, enable the "Analyze generated code" setting inside **Project > Administration > General Settings > VB.NET**. By default, tool-generated code files are skipped from analysis.
 
-## Known Limitations
-Currently an error will be thrown when an issue is raised on a line of code containing the following pattern `\s+error\s*:` (i.e. one or more spaces, the string 'error', zero or more spaces and a ':' ) . This is a well known problem on the Microsoft side (see [issue](https://github.com/dotnet/roslyn/issues/5724/)). In order to work around this problem, analysis will skip issues reported on any line where the pattern is detected.
+Generated code detection is based on file name, special comments and attributes. Currently recognized values are in [GeneratedCodeRecognizer.cs](https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/src/SonarAnalyzer.Common/Helpers/GeneratedCodeRecognizer.cs).
 
 ## Scanner compatibility
 
 To analyze VB.NET code, you need to use the [SonarScanner for .NET](/analysis/scan/sonarscanner-for-msbuild/) version 4.x or newer.
 
+## Exclusions
+
+Files excluded from analysis should be set in a project configuration. This exclusion still performs the analysis on the file and filters final results.
+
+As an alternative, `.editorconfig` file can be used to disable analysis for a specific rule on a file or directory. This can solve performance problems on large files.
+
+```
+[Path/File.vb]
+dotnet_diagnostic.Sxxx.severity = none
+```
+
+## Known Limitations
+
+Currently an error will be thrown when an issue is raised on a line of code containing the following pattern `\s+error\s*:` (i.e. one or more spaces, the string 'error', zero or more spaces and a ':' ) . This is a well known problem on the Microsoft side (see [issue](https://github.com/dotnet/roslyn/issues/5724/)). In order to work around this problem, analysis will skip issues reported on any line where the pattern is detected.
+
 ## Related Pages
 * [Importing External Issues](/analysis/external-issues/) (VSTest, NUnit, MSTest, xUnit)
 * [Test Coverage & Execution](/analysis/coverage/) (Visual Studio Code Coverage, dotCover, OpenCover, NCover 3)
```

Comment:
how about also adding links to our guide: https://community.sonarsource.com/t/9871 and https://community.sonarsource.com/t/37151

Reply:
I've added the second one. The first one is not so related to the analyzer itself, but rather about .NET tooling around.

Reply:
ok, anyway the troubleshooting guide has a link to them